examples/custom-field-react-starter/DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,169 @@
+# 🚀 Developer Guide: Creating a Custom Field with MyWave UI
+
+## 1. Clone the Starter Template
+Begin by cloning the custom field starter template:
+
+```
+git clone https://github.com/mywave-me/mywave-ui-examples.git
+cd mywave-ui-examples
+```
+
+### Prerequisites
+Ensure you have the following installed:
+
+- Node.js (v20.12.2)
+- pnpm (v9.0.6)
+
+If not, run the setup script:
+
+```
+./scripts/preinstall.sh
+```
+
+## 2. Install Dependencies
+Navigate to the project directory and install the required dependencies:
+
+```
+cd examples/custom-field-react-starter
+pnpm install
+```
+
+## 3. Explore the Starter Files
+Open the project in your preferred code editor. Familiarize yourself with the following key files:
+
+`src/myCustomField.tsx`: The main component where the custom field is defined.
+
+`src/main.tsx`: The entry point that renders the CustomField component.
+
+`index.html`: The HTML template that includes the root div (`<div id="root"></div>`).
```

Comment:
This is more project `README.md` stuff. I guess the main question is what problem is this guide trying to solve and what should go here versus in the `README.md`? As seems like there could be a bit of overlap